packages/cashier/src/Containers/withdrawal.jsx

```diff
@@ -19,6 +20,11 @@ const WithdrawalSideNote = () => {
             i18n_default_text='Do not enter an address linked to an ICO purchase or crowdsale. If you do, the ICO tokens will not be credited into your account.'
             key={0}
         />,
+        <Localize
+            i18n_default_text='It may take up to <0>6 confirmations</0> for your funds to be reflected in your destination wallet.'
```

Comment:
Don't we want to localize the `confirmations` text as well?

Reply:
This won't localize it? How would you do it with the bold text?

Reply:
No, since anything in a key is a `variable` and would not be part of the strings that will be translated on Crowdin.

You can pass the `<strong>` tags as a variable and wrap it around the `confirmations` text.

Reply:
Ah my bad, this should work. You're already using the `Text` component as the variable. 
Please ignore. 🙏🏻